randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -503,7 +504,7 @@ Extra - room_id: [14]
   > Door to Aquarium Tank
       Trivial
   > Other to Cargo Hold to Sector 5 (ARC)
-      Trivial
+      Morph Ball
```

Comment:
Oh whoops, that might make a bunch of the general fixes unnecessary - I can clear those out

Reply:
That's okay, no need to change. These can make the database clearer to newer folks since the docknode weaknesses aren't exposed to regular viewers and they may think it's flawed logic (but really it isn't).